backend/src/providers/accounts.ts

```diff
@@ -21,7 +20,7 @@ export const isAccountIndexed = async (accountId: string): Promise<boolean> => {
 export const getAccountsList = async (
   limit: number,
   lastAccountIndex: number | null
-): Promise<AccountListInfo[]> => {
```

Comment:
Because the type is guarded by the `Kysely` lib in `queries.ts`, there is no need to try to type it once more here.
I actually plan to move queries to the actual functions they belong two, I don't see the need to keep them apart anymore, it just adds hussle.